cc/gtp_player.cc

```diff
@@ -103,9 +103,9 @@ GtpPlayer::Response GtpPlayer::DispatchCmd(
     return HandleEcho(cmd, args);
   } else if (cmd == "final_score") {
     return HandleFinalScore(cmd, args);
-  } else if (cmd == "gamestate" || cmd == "mg_gamestate") {
+  } else if (cmd == "gamestate" || cmd == "gamestate") {
```

Comment:
are these different in some exciting and subtle way that i cannot detect with my mortal eyes?

Reply:
no, no, I'm just stupid